samples/unit-tests/chart/linewidth/demo.js

```diff
@@ -0,0 +1,93 @@
+QUnit.test('normal:lineWidth and hover:lineWidthPlus #4035', function (assert) {
```

Comment:
Move to `unit-tests/series-line/linewidth` instead, as this is a property of line-based series.